openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -4353,9 +4368,24 @@ def create_force(self, system, topology, **kwargs):
                 atom_indices = (
                     charge_increment_match.environment_match.topology_atom_indices
                 )
-                charge_increment = charge_increment_match.parameter_type
+                charge_increments = copy.deepcopy(charge_increment_match.parameter_type.charge_increment)
+
+                # If we've been provided with one less charge increment value than tagged atoms, assume the last
+                # tagged atom offsets the charge of the others to make the chargeincrement net-neutral
+                if len(charge_increments) - len(atom_indices) == -1:
+                    charge_increment_sum = 0. * unit.elementary_charge
+                    for ci in charge_increments:
+                        charge_increment_sum += ci
+                    charge_increments.append(-charge_increment_sum)
+                elif len(charge_increments) - len(atom_indices) == 0:
+                    pass
+                else:
+                    raise SMIRNOFFSpecError(f"Trying to apply chargeincrements {charge_increment_match.parameter_type} "
```

Comment:
This exception _and_ the one at L4231 are both raised by tests, according to codecov. I found this counter-intuitive at first, but now I see in one of the tests that you modified a handler after construction achieve this broken state. Nice!

Reply:
Yeah, the other exception/test is a _bit_ forced, but I had already gone through the work of making a new exception, and I think it set the record for exception name length, so I opted to keep it.